tests/e2e/page-router.test.ts

```diff
@@ -157,12 +157,15 @@ test.describe('Simple Page Router (no basePath, no i18n)', () => {
 
         const fallbackWasServed =
           useFallback && headers1['cache-status'].includes('"Next.js"; fwd=miss')
-        expect(headers1['netlify-cache-tag']).toBe(
-          fallbackWasServed ? undefined : `_n_t_${encodeURI(pagePath).toLowerCase()}`,
-        )
+        if (!fallbackWasServed) {
+          expect(headers1['netlify-cache-tag']).toBe(`_n_t_${encodeURI(pagePath).toLowerCase()}`)
+        }
```

Comment:
fallback does get cache tag with newest canary, but as long as it's not cacheable, we don't actually care about cache tag